opentelemetry-otlp/tests/grpc_build.rs

```diff
@@ -43,15 +43,14 @@ fn build_content_map() -> HashMap<PathBuf, String> {
     let mut map = HashMap::new();
     let dict =
         std::fs::read_dir("src/proto/grpcio").expect("cannot open dict of generated grpc files");
-    for entry in dict {
-        if let Ok(entry) = entry {
-            map.insert(
-                entry.path(),
-                std::fs::read_to_string(entry.path()).unwrap_or_else(|_| {
-                    panic!("cannot read from file {}", entry.path().to_string_lossy())
-                }),
-            );
-        }
+
+    for entry in dict.into_iter().flatten() {
```

Comment:
Maybe rewrite this to collect() while you're at it? Potentially more efficient to allocate the correct size immediately.

Reply:
Sounds good to me. But I thought the compiler could allocate the correct size with for loop also?